package.json

```diff
@@ -34,19 +34,25 @@
   },
   "homepage": "https://github.com/mirainc/schedule-utilities#readme",
   "devDependencies": {
-    "babel-cli": "^6.26.0",
+    "@types/jest": "^22.2.3",
+    "@types/moment-timezone": "^0.5.12",
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-stage-1": "^6.24.1",
+    "babel-runtime": "^6.26.0",
     "jest": "^22.0.6",
+    "rrule": "^2.6.0",
```

Comment:
We import `rrule` here purely to supply typings for `rrule-alt` in [rrule-alt.d.ts](https://github.com/mirainc/schedule-utilities/pull/5/files#diff-609572308fb01619867df9ad1c35d551)